src/frontend/src/lib/components/transactions/AllTransactions.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import TransactionsPlaceholder from '$lib/components/transactions/TransactionsPlaceholder.svelte';
+	import { SLIDE_DURATION } from '$lib/constants/transition.constants';
+	import { enabledTokens } from '$lib/derived/tokens.derived';
+	import type { UnifiedTransactionUi } from '$lib/types/transaction';
+	import {
+		isNetworkIdBTCMainnet,
+		isNetworkIdEthereum,
+		isNetworkIdICP
+	} from '$lib/utils/network.utils';
+
+	let transactions: UnifiedTransactionUi[];
+	$: transactions = $enabledTokens.reduce<UnifiedTransactionUi[]>(
```

Comment:
This will be extracted to a utility with a test in a later PR, correct?

Reply:
correct! for now it is just a show case